python-packages/contract_addresses/src/zero_ex/contract_addresses/__init__.py

```diff
@@ -66,16 +66,17 @@ class NetworkId(Enum):
 
 NETWORK_TO_ADDRESSES: Dict[NetworkId, ContractAddresses] = {
     NetworkId.MAINNET: ContractAddresses(  # nosec
-        erc20_proxy="0x2240dab907db71e64d3e0dba4800c83b5c502d4e",
-        erc721_proxy="0x208e41fb445f1bb1b6780d58356e81405f3e6127",
+        erc20_proxy="0x95e6f48254609a6ee006f7d493c8e5fb97094cef",
+        erc721_proxy="0xefc70a1b18c432bdc64b596838b4d138f6bc6cad",
         zrx_token="0xe41d2489571d322189246dafa5ebde1f4699f498",
         ether_token="0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-        exchange="0x4f833a24e1f95d70f028921e27040ca56e09ab0b",
-        asset_proxy_owner="0x17992e4ffb22730138e4b62aaa6367fa9d3699a6",
-        forwarder="0x5468a1dc173652ee28d249c271fa9933144746b1",
-        order_validator="0x9463e518dea6810309563c81d5266c1b1d149138",
-        coordinator_registry="0x45797531b873fd5e519477a070a955764c1a5b07",
-        coordinator="0x25aae5b981ce6683cc5aeea1855d927e0b59066f",
+        exchange="0x080bf510fcbf18b91105470639e9561022937712",
+        asset_proxy_owner="0xdffe798c7172dd6deb32baee68af322e8f495ce0",
+        forwarder="0xdc4587cb17d2a1829512e2cfec621f8066290e6a",
+        order_validator="0xa09329c6003c9a5402102e226417738ee22cf1f2",
+        # TODO(gene): Update these addresses once the coordinator contracts are re-deployed
```

Comment:
These need to be added before we merge, otherwise, the address runtime validators will raise an exception.